layouts/shortcodes/release-branches.html

```diff
@@ -0,0 +1,45 @@
+<h2 id="detailed-release-history-for-active-branches">Detailed Release History for Active Branches</h2>
+{{ range $releaseInfo := .Site.Data.releases.schedule.schedules }}
+{{- $version := printf "%.2f" $releaseInfo.release -}}
+{{- $dataIdVersion := replace ( printf "%.2f" $releaseInfo.release ) "." "-" | anchorize -}}
+
+<h3 id="{{ $dataIdVersion }}">{{ $version }}</h3>
+<p>
+    Next patch release is <strong>{{ $releaseInfo.next.release }}</strong>. {{ $releaseInfo.next.note }}
```

Comment:
To make this work, you can load a _format string_ using the `T` localization function, and then format ``$releaseInfo.next.release` into that string.
Document to the localization teams that the string you define is a format string.